docs/multicluster/architecture.md

```diff
@@ -201,3 +200,25 @@ be created as a ResourceExport in the leader cluster, and the resource
 export/import pipeline will ensure member clusters receive this ACNP spec
 to be replicated. Each member cluster's Multi-cluster Controller will then
 create an ACNP in their respective clusters.
+
+## Antrea Multi-cluster Traffic Modes
```

Comment:
I feel we should remove "Multi-cluster", as it is Antrea modes, not Multi-cluster modes.

Reply:
Removed